frontend/src/metabase/containers/SaveQuestionModal.jsx

```diff
@@ -87,7 +87,8 @@ export default class SaveQuestionModal extends Component {
           : "",
       description: card.description || "",
       collection_id:
-        card.collection_id === undefined
+        card.collection_id === undefined ||
+        (originalCard != null && !originalCard.can_write)
```

Comment:
I think it might be worth extracting to a constant with a readable name. Needed a moment to parse that